--- tokyocabinet-ruby-1.31/tokyocabinet.gemspec.orig	2010-09-20 01:12:57.000000000 +0200
+++ tokyocabinet-ruby-1.31/tokyocabinet.gemspec	2017-02-13 18:28:58.142358297 +0100
@@ -3,7 +3,7 @@
 spec = Gem::Specification.new do |s|
   s.name = "tokyocabinet"
   s.version = "1.31"
-  s.author "FAL Labs"
+  s.author = "FAL Labs"
   s.email = "info@fallabs.com"
   s.homepage = "http://fallabs.com/tokyocabinet/"
   s.summary = "Tokyo Cabinet: a modern implementation of DBM."
